### ISSUE TITLE
vmui: step override isn't reflected in URL

### ISSUE THREAD

**Issue by hagen1778** [VictoriaMetrics/VictoriaMetrics#3513]
**Describe the bug**
When overriding step param the URL doesn't change.

**To Reproduce**
Override the step param, execute the query and check URL:
<img width="673" alt="image" src="https://user-images.githubusercontent.com/2902918/208706556-82ecbbee-21c3-4d9f-861e-e885d4b1d6ed.png">

**Version**
The line returned when passing `--version` command line flag to the binary. For example:
```
$ ./bin/victoria-metrics --version
victoria-metrics-20221220-151843-heads-master-0-g18e55d14c
```

**Comment by valyala** [comment]
See the related bug - https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3529#issuecomment-1364366922

**Comment by valyala** [comment]
The bug has been fixed in d2f89b55b73e229a40eae597a68a99f6dd543e62 . This commit will be included in the next release of VictoriaMetrics.

**Comment by valyala** [comment]
The bugfix has been included in [v1.86.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) - now `vmui` should properly update `step` query arg after it is changed. Closing the issue as fixed.
